packages/esm-bed-management-app/src/bed-tag/bed-tags-admin-form.component.tsx

```diff
@@ -110,16 +110,16 @@ const BedTagsAdministrationForm: React.FC<BedTagAdministrationFormProps> = ({
               />
             )}
           </Stack>
-        </ModalBody>
-        <ModalFooter>
-          <Button onClick={() => onModalChange(false)} kind="secondary">
-            {t('cancel', 'Cancel')}
-          </Button>
-          <Button disabled={!isDirty} type="submit">
-            <span>{t('save', 'Save')}</span>
-          </Button>
-        </ModalFooter>
-      </Form>
+        </Form>
+      </ModalBody>
+      <ModalFooter>
+        <Button onClick={() => onModalChange(false)} kind="secondary">
+          {getCoreTranslation('cancel', 'Cancel')}
+        </Button>
+        <Button disabled={!isDirty} onClick={handleSubmit(onSubmit, onError)}>
```

Comment:
I don't think we need the submit handler wired up both here and in the `Form` tag. The handler in the `Form` tag should be sufficient.